webapp/package.json

```
@@ -65,5 +65,10 @@
     "ts-node": "^10.9.1",
     "tsconfig-paths": "^4.1.0",
     "typescript": "^4.8.3"
+  },
+  "exports": {
+    "./query/react": "./dist/query/react/index.js",
+    "./query": "./dist/query/index.js",
```

My suggestion was to add the comment on the `package.json` so that other maintainers understand the historical reason for the workaround in the future. 😄
```suggestion
  "//" : "This is a workaround related to Typescript's node module resolution, which is used by RTK query.",
  "exports": {
    "./query/react": "./dist/query/react/index.js",
    "./query": "./dist/query/index.js",
```